apps/user-profile/src/components/modules/UserOnboardingModal/OnboardingCard/OnboardingCard.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Typography } from '@devlaunchers/components/components/atoms'
+import { OnboardingCardContainer, PicWrapper, TextWrapper, CheckedWrapper, CheckedSVG, IconImg } from './StyledOnboardingCard'
+export default function OnboardingCard({
```

Comment:
Please can you add a component comment describing what the component does: 
![Screenshot 2023-04-23 at 19 37 40](https://user-images.githubusercontent.com/125300151/233858517-03a2ba68-9e56-4220-87bd-7fc57302e41c.png)